go.mod

```diff
@@ -21,10 +21,10 @@ require (
 	go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp v0.46.0
 	go.opentelemetry.io/contrib/propagators/b3 v1.20.0
```

Comment:
Should this be 1.21.0?

Reply:
its 21 right?

Reply:
https://github.com/open-telemetry/opentelemetry-go-contrib/releases/tag/propagators%2Fb3%2Fv1.21.0 otel go and contrib versions are all over the place :( eg for contrib https://github.com/open-telemetry/opentelemetry-go-contrib/releases/tag/v1.21.0 - `Release v1.21.0/v0.46.0/v0.15.0/v0.1.0` Depending on the module it can be one of those

Reply:
yes, different packages follow different versioning :(